hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/bucket/AddAclBucketHandler.java

```diff
@@ -92,8 +92,8 @@ public Void call() throws Exception {
     boolean result = client.getObjectStore().addAcl(obj,
         OzoneAcl.parseAcl(acl));
 
-    System.out.printf("%s%n", JsonUtils.toJsonStringWithDefaultPrettyPrinter(
-        JsonUtils.toJsonString("Acl set successfully: " + result)));
+    System.out.printf("%s%n", result ? "ACL set successfully" : "ACL not set");
```

Comment:
False means Acl passed is already existing, in this case we return false. So message `ACL not set `is meaningful here or do we need to reword this?